odoo/models.py

```diff
@@ -3360,7 +3360,7 @@ def _write(self, vals):
                 _logger.warning('Field %s is deprecated: %s', field, field.deprecated)
 
             if field.column_type:
-                if single_lang or not (has_translation and field.translate is True):
```

Comment:
This may silently break cases where the business logic relied on the previous "unspecified" behavior. Apparently it was the case in web_studio, so creation/update of new inherited views does not work anymore in non-english languages there.
We could consider that web_studio should force `lang=None` or use `arch_base` instead of `arch`, but this can similarly affect other "legit" cases. Shouldn't we at least log a warning when the unspecified case occurs (i.e. write to an XML translatable field in a non-master lang), and see if we detect anything in the tests?

Reply:
This may need to be patched in 11.0, if you backported this patch already. /cc @kebeclibre